### PR TITLE
Removing margin-bottom on header component

### PR DESCRIPTION
**Pull request by ldabiralai**
This is in relation to #43

### CODE REVIEW

**Comment by alexflorisca** [comment]
:+1:

**Comment by peter-mouland** [comment]
:+1:

File: server/index.hbs [hunk 1]
```diff
@@ -443,26 +443,28 @@
...
-        <header>
-          <div class="header-circle">
-            <div class="header-circle-background"></div>
-            <i class="header-circle-icon icon-content-passes"></i>
-          </div>
-          <h1 class="header-heading">Lorem Ipsum</h1>
-          <p class="header-subheading">Lorem ipsum dolor sit amet.</p>
-        </header>
+      <div class="header-container">
+        <div class="header_green">
```
Comment (peter-mouland) [inline server/index.hbs]: is this new div part of the recommended way of coding headers?
if so, should the `server/styles/modules/header.scss` be part of the src styles?

**Comment by ldabiralai** [comment]
It's not part of the recommended way to code headers, I'm not sure how we can make that clear?

The header container's purpose it only for styling the divs under it with margin-bottom to make the styleguide docs look a little nicer, as suggested by @ariley90 when he was reviewing the PR.

**Comment by alexjfno1** [comment]
@peter-mouland @ldabiralai Are you talking about the header-container class only being needed in the docs page in order to make it look nice on the page.

Could we not prefix all classes which are for the presentation of the docs with `docs__<class-name>` so that when people are inspecting the source they know not to use that class?

**Comment by ldabiralai** [comment]
Good idea @alexjfno1 

Maybe we should add this somewhere so that people know not to use these classes? Or should we assume that people will understand that?

I've made the change for my PR for now though.

**Comment by nowarkdev** [comment]
@alexjfno1 :+1:

**Pull request merged**